### PR TITLE
updated global distributed blog with wash up

### PR DESCRIPTION
**Pull request by brooksmtownsend**
This PR updates the globally distributed blog with `wash up`, providing an overall better experience for creating a superclustered lattice. There will be more content around superclusters coming but another post that showed the experience with `wash up` would essentially be a duplicate of this post.

Planned for release the week of Oct 10th

PR deploy failed as an anomaly, re-running was fine https://deploy-preview-32--dreamy-golick-5f201e.netlify.app/blog/globally_distributed_webassembly_applications_with_wasmcloud_and_nats/

### CODE REVIEW

**Review by thomastaylor312 — COMMENTED** [review]
This should be good to go once you want to merge. Will give final ✅ once you switch this from draft

**Comment by brooksmtownsend** [comment]
@thomastaylor312 / @autodidaddict ready for review here

**Review by autodidaddict — APPROVED** [review]

**Pull request merged**
